lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction.js

```diff
@@ -0,0 +1,138 @@
+const clone = require('../../util/clone');
+const mixedNumber = require('./mixedNumber');
+
+const ChangeTypes = require('../../ChangeTypes');
+const Node = require('../../node');
+
+// Converts a mixed number to an improper fraction
+// e.g. 1 2/3 -> 5/3
+function convertMixedNumberToImproperFraction(node) {
+  if (!mixedNumber.isMixedNumber(node)) {
+    return Node.Status.noChange(node);
+  }
+
+  const substeps = [];
+  let newNode = clone(node);
+
+  // e.g. 1 2/3
+  const wholeNumber = mixedNumber.getWholeNumber(node); // 1
+  const numerator = mixedNumber.getNumerator(node); // 2
+  const denominator = mixedNumber.getDenominator(node); // 3
+  const isNegativeMixedNumber = mixedNumber.isNegativeMixedNumber(node);
+
+  // STEP 1: Convert to unsimplified improper fraction
+  // e.g. 1 2/3 -> ((1 * 3) + 2) / 3
```

Comment:
I don't disagree that it's less clear than your approach, but I think it's a pretty common standard.